### DISCUSSION TITLE
【委托】很急！！！

### DISCUSSION THREAD

**Discussion by ysd1911** [share121/inter-knot#2737] [其他]
![20240707153007](https://github.com/user-attachments/assets/9840f0af-faec-46c8-b10a-956cdfa36154)
有没有人来帮我去空洞取样东西，旧都沦陷时跑的太急了，家里代代传下来的失落技术的实验数据留在莱姆尼安空洞里了，求位高手绳匠帮我取回，价钱好说，很急！希望数据还没有被以太侵蚀
（注：莱姆尼安空洞极度危险，没有经验的绳匠不要接，最好是有经验并且业务能力强的绳匠来帮我！）

**Comment by lllliufeng** [comment]
我tm来辣！

**Comment by YuShaozhu** [comment]
这么快的吗

**Comment by MrDucker** [comment]
这个委托我接了<img src="https://bbs-static.miyoushe.com/static/2023/11/16/53069b115197426eff86537576a35d75_2306411376722926481.png" alt="外卖员邦布-出发" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/11/16/810dd11a4d16759f16ca078596aab6b1_422007669816866994.png" alt="哲-让我想想" width="75" >

**Reply by ysd1911** [reply]
感谢这位绳匠，事成之后必有重谢，地址发给你了，空洞距离新艾利都有段距离，到时候你拿着萝卜来就好了

**Comment by HKFDBHZN** [comment]
研究数据？嘿嘿嘿...

**Comment by SKAIZH** [comment]
我是法厄同

**Comment by Acacialpz** [comment]
我才是法厄同，v我五十，我来助你

**Comment by demolute** [comment]
我是警方，都老实点！

**Reply by ysd1911** [reply]
这位先生您好，请不要在绳网发送这些扰乱人心的言论，如果治安局能满足每个人的需求，我们当然也不会在这里相遇，所以，请你收起你那无力的话语，

**Reply by Acacialpz** [reply]
啊哈，太有乐子了0.o
被制裁了bro

**Reply by shan-dgha** [reply]
这位警察，在绳网上我们甚是怀疑你的业务能力。请出示证明！

**Comment by aizai1173** [comment]
开拓者能接嘛？

**Comment by baiyangQAQ** [comment]
这个委托我让舰长来接<img src="https://bbs-static.miyoushe.com/static/2024/07/31/de6cf640b821a169d3cfb52621cec21b_242979188329737974.png" alt="椒丘 辣" width="75" >
